### DISCUSSION TITLE
Being able to optionally disable logging of "GET /status" in the access logs

### DISCUSSION THREAD

**Discussion by subnetmarco** [Kong/kong#7276] [Ideas and feature requests]
Kong provides a `GET /status` endpoint to check for the status of Kong, which should be optionally disabled since it creates lots of noise in the access logs when a client is making frequent requests to it.

**Comment by sudoapt-getclean** [comment]
We have Kong running in kubernetes with datadog and an ALB load-balancing(multi-region so effectively three). All 5 are hitting this endpoint to generate 50 requests per minute per kong instance (we have 2)
50 × 2(hosts) × 60(minutes) × 24(hours) × 30(days) = 4_320_000 
so that's 4 million log lines that we would like to turn off.

**Comment by dweber019** [comment]
We would like to have this too as the status log entries are irrelevant for us and just generates costs.

**Comment by zackery-parkhurst** [comment]
Its even more than this if you are running on kubernetes. As then you have the probes hitting the endpoints as well. Generating millions of logs. 

Really sad that both the issue that started this discussion and this discussion has been completely ignored for what 5 years now.

**Reply by fffonion** [reply]
For healthcheck and probes, it's recommened to use the status endpoint instead of admin endpoint, and a seperate log location can be configured. https://docs.konghq.com/gateway/latest/reference/configuration/#status_access_log

The status port is configurable through https://docs.konghq.com/gateway/latest/reference/configuration/#status_listen and default to 8100.

Do not open admin port even it's in your internal k8s cluster without protection. Use status endpoint as it's read-only.

**Comment by hanshuebner** [comment]
Kong Gateway does not provide a way to filter individual access log entries, but you can use a custom plugin to generate the access log, and implement your filtering logic there.  Alternatively, you can use a [custom nginx template](https://docs.konghq.com/gateway/latest/reference/nginx-directives/#custom-nginx-templates) and configure [conditional logging](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#enabling-conditional-logging) there:

In the http block:
```
    # Define a map to set a variable based on the request URI
    map $request_uri $loggable {
        default 1;
        ~^/status 0;
    }
```
In the server block:
```
   # Set the access log with the conditional variable
        access_log /var/log/nginx/access.log main if=$loggable;
    }
```

**Reply by 23henne** [reply]
Can we use it with environment injection as well? I currently fail to do so.
`KONG_NGINX_HTTP_MAP: $request_uri $keeplog { ~/health 0; default 1; }`

This leads to "unexpected ';'" error.
